requirements/frozen.txt

```diff
@@ -259,6 +259,7 @@ pyperclip==1.7.0
 pytesseract==0.3.0
 pytest==5.2.1
 pytest-polarion-collect==0.23.1
+pytest-mock==2.0.0
```

Comment:
This requirement needs to be in the template_non_imported.txt file, as its not directly imported but the fixture is indeed used in the unit test.

When adding requirements, this can be done by specifying
```
miq requirement add --extra pytest-mock
```

Reply:
I put the line in template_non_imported.txt as the last line. Let me know if it is the wrong place. Also, I need to keep line 262 above?

Reply:
thanks, and correct, still needs to be in frozen.

the requirement script will install both template_scanned.txt and template_non_imported.txt, with the constraints file, and then generate this frozen file from the resulting venv packages.